site/content/tutorial/02-reactivity/04-updating-arrays-and-objects/text.md

```diff
@@ -13,15 +15,15 @@ function addNumber() {
 }
 ```
 
-But there's a more idiomatic solution:
+You can also get a similar result using the ES6 spread syntax:
 
 ```js
 function addNumber() {
 	numbers = [...numbers, numbers.length + 1];
 }
 ```
 
-You can use similar patterns to replace `pop`, `shift`, `unshift` and `splice`.
+The same rule applies to array methods such as `pop`, `shift`, `splice` and to objects methods such as `Map.set`.
```

Comment:
should include "and" before the last item in a list. maybe we should have multiple examples of object methods as well?
```suggestion
The same rule applies to array methods such as `pop`, `shift`, and `splice` and to objects methods such as `Map.set`, `Set.add`, etc.
```

Reply:
@stephanietuerk Can you make the changes? (Github is annoying when it comes to making PRs to other forks)